lib/org/chromium/webidl/ast/Named.js

```diff
@@ -7,6 +7,11 @@ foam.CLASS({
   package: 'org.chromium.webidl.ast',
   name: 'Named',
 
+  // The actual literal is stored in name.literal... will this be an issue?
+  // Also, Operations nodes are Named... but they have optional identifiers...
+  // Thus, name === null for some nodes...
+  ids: ['name'],
```

Comment:
`Operation` nodes implements `Named`, however, they have optional identifiers (as specified by the grammar). In these cases, `name === null` and issues may arise in the future with regards to this.